ssp.md

```diff
@@ -31,11 +31,11 @@ Each of the models produced results for between three and five of the SSPs, with
 
 Official SSP data, as planned for use in [CMIP6](http://www.geosci-model-dev.net/9/3461/2016/), must be downloaded from the official [SSP database](https://secure.iiasa.ac.at/web-apps/ene/SspDb/dsd?Action=htmlpage&page=about).
 
-The GCAM data for the SSP4 and the GCAM representation of all other SSPs (Calvin et al., 2017) are being made available to interested researchers for scenario analysis and comparison. This data is available at ***INSERT LINK***
+The GCAM data for the SSP4 and the GCAM representation of all other SSPs (Calvin et al., 2017) are being made available to interested researchers for scenario analysis and comparison. 
 
 ## Implementation in GCAM
 
-To implement the SSPs within GCAM, we use the demographic and economic assumptions developed by Lutz et al. (2015) and Dellink et al. (2015) in combination with technology and policy assumptions derived from the SSP narratives (O’Neill et al. 2015). Table 1 provides a qualitative summary of these parameters; quantitative details are provided in subsequent tables. In general, we have altered parameters related to energy supply (e.g., the capital cost of new power plants, technical change on extraction costs), energy demand (e.g., preferences for traditional bioenergy), agriculture (e.g., agricultural productivity growth), and policy (e.g., start years for carbon policies). To implement the SSP storylines within GCAM, we quantified “high”, “medium”, and “low” for each of the variables listed in Table 2. Those quantifications are provided in Tables 3 through Table 8.
+To implement the SSPs within GCAM, we use the demographic and economic assumptions developed by Lutz et al. (2015) and Dellink et al. (2015) in combination with technology and policy assumptions derived from the SSP narratives (O’Neill et al. 2015). Table 2 provides a qualitative summary of these parameters; quantitative details are provided in subsequent tables. In general, we have altered parameters related to energy supply (e.g., the capital cost of new power plants, technical change on extraction costs), energy demand (e.g., preferences for traditional bioenergy), agriculture (e.g., agricultural productivity growth), and policy (e.g., start years for carbon policies). To implement the SSP storylines within GCAM, we quantified “high”, “medium”, and “low” for each of the variables listed in Table 2. Those quantifications are provided in Tables 3 through Table 8.
```

Comment:
I think the form of "Table" should agree in the last sentence.  So, either "Tables 3-8" or "Table 3 through Table 8".